### ISSUE TITLE
Theme watch/upload not working since v0.7.0

### ISSUE THREAD

**Issue by dankiv** [Shopify/themekit#517]
### What happened
After updating themekit from 0.6.12 to latest (v.0.7.5) it's not uploading changes. Tried all other versions > 0.6.12, but it doesn't work.  v.0.6.12 works fine after revert. 

"theme watch" command is showing:
10:03:53 [development] Watching for file changes on host  ****

But nothing triggers after files changes

### What I expected to happen
Files need to be uploaded after changes

### Steps to reproduce

1. Run theme watch
2. Change file
3. No changes triggered

### My Environment

OS (including version): Windows 10
Themekit version:  0.7.5
Ignore Files(if any):

**Comment by chuckreynolds** [comment]
fwiw on a fresh install of `ThemeKit 0.7.5 darwin/amd64` on macOS my `watch` is working as intended.

**Comment by tanema** [comment]
Can you give me an example of the filepaths you are changing?

**Comment by tejashweta** [comment]
Did anyone resolve it? I'm facing the exact same error.

**Comment by tanema** [comment]
@tejashweta I saw you commenting on https://github.com/shopify-graveyard/shopify_theme/issues/158 which is a dead project. Are you still using the ruby version of themekit? If so it is no longer supported and you should move to a new version of themekit. This issue is wildly out of date as well. please update

**Comment by Bernardinevw89** [comment]
In case this solves it for anyone else:

I had the same issue and only later on realized that we changed our file structure slightly, I then discovered if you add the working directory path to config.yml it works again. Our config.yml file is always in the src directory, we then took all of the standard theme files and placed them in a folder called "theme" within the /src folder so config.yml was still in the src folder and obviously couldn't locate the theme files so here's what I changed in config.yml https://share.getcloudapp.com/bLuOWdl8

I went through the ThemeKit troubleshooting doc but this wasn't mentioned there, think there should be a step added that tells you to ensure that the working directory is correct.

Hope this helps!
